### ISSUE TITLE
[bug] toolbar menus not showing on Linux

### ISSUE THREAD

**Issue by slightlyskepticalpotat** [dogecoin/dogecoin#3564]
# Bug Report

**Current behavior:**
When Dogecoin Core is launched on a non-primary display, toolbar menus don't show when "File", "Settings", "Help" are clicked. In the image below, "Settings" is being clicked.
![image](https://github.com/dogecoin/dogecoin/assets/31290848/8d0194f2-4286-4665-b7d2-77991f4b15f7)

**Expected behavior**
The menus should be shown. This is the behaviour when Dogecoin Core is opened or manually moved to the primary window.
![image](https://github.com/dogecoin/dogecoin/assets/31290848/08910e23-b955-4077-b749-4173fea10e6f)

**Steps to reproduce:**

1. Download official binaries for Linux x86_64
2. Launch dogecoin-qt when at least two monitors are connected
3. Manually drag dogecoin-qt over to the secondary window
4. Click one of the options in the toolbar

**Dogecoin Core version**
Issue exists on official 1.14.7 and 1.14.6 binaries, possibly other versions as well.

**Machine specs**
- OS: Ubuntu 22.04.4 LTS x86_64 (Wayland, GNOME)
- CPU: AMD Ryzen 5 5500U
- RAM: 16GB
- Disk size: 256GB SSD

**Extra information**
__This is probably an issue with Qt or GNOME, just creating this bug so I can investigate and update/see if anyone else has experienced it.__

**Comment by patricklodder** [comment]
I've seen this on macOs too.

Perhaps this is to do with on which actions [`BitcoinGUI::setWalletActionsEnabled`](https://github.com/dogecoin/dogecoin/blob/a6d12201363b0eb4ccf854b609fc6e98958b1443/src/qt/bitcoingui.cpp#L589-L606) is called?

**Comment by slightlyskepticalpotat** [comment]
@patricklodder do you usually see it on your local builds? I definitely see it on official binaries but haven't been able to reproduce in any local builds.

**Comment by patricklodder** [comment]
> do you usually see it on your local builds?

I've tested this and I cannot reproduce it with a recent Qt. I think that ultimately, the solution is to continue working on the Qt6 integration and get rid of 5.x

**Comment by slightlyskepticalpotat** [comment]
Makes sense, I'll keep an eye out to see if this persists after the move to Qt6!
